cdc/log.cc

```diff
@@ -410,7 +410,6 @@ static schema_ptr create_log_schema(const schema& s, std::optional<utils::UUID>
                         return type.freeze();
                     }
                 ));
-                type = type->freeze();
```

Comment:
@elcallio is it ok to remove this freeze? I think it should be but want to make sure it's not too tricky for me.

Reply:
`visit` returns a frozen type for each type.